hopp/simulation/hybrid_simulation.py

```diff
@@ -337,24 +345,38 @@ def set_om_costs_per_kw(self, pv_om_per_kw=None, wind_om_per_kw=None,
         #     raise ValueError(f"Length of yearly om cost per kw arrays must be equal. Some lengths of om_per_kw values are different from others: {om_lengths}")
         if pv_om_per_kw and self.pv:
             self.pv.om_capacity = pv_om_per_kw
+            if pv_om_per_mwh:
```

Comment:
Totally agree that it would be great to do some tech abstraction so it doesn't have to get so long but there are several places in `HybridSimulation` where I think that abstraction would be valuable. I don't think I have time right now to do that but I've made an issue about it